Update fuzzy.md (#7842)
opensearch-project/documentation-website: _query-dsl/term/fuzzy.md

# Fuzzy query

A fuzzy query searches for documents containing terms that are similar to the search term within the maximum allowed [Damerau–Levenshtein distance](https://en.wikipedia.org/wiki/Damerau–Levenshtein_distance). The Damerau–Levenshtein distance measures the number of one-character changes needed to change one term to another term. These changes include:

- Replacements: **c**at to **b**at
- Insertions: cat to cat**s**
- Deletions: **c**at to at
- Transpositions: **ca**t to **ac**t

> A fuzzy query creates a list of all possible expansions of the search term that fall within the Damerau-Levenshtein distance (by the way, if you set the `transpositions` field equal to `false`, your search will use the classical [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance)). You can specify the maximum number of such expansions in the `max_expansions` field. Then it searches for documents that match any of the expansions.

Reviewer: ```suggestion
A fuzzy query creates a list of all possible expansions of the search term that fall within the Damerau-Levenshtein distance. You can specify the maximum number of such expansions in the `max_expansions` field. The query then searches for documents that match any of the expansions. If you set the `transpositions` parameter to `false`, then your search will use the classic [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance). 
```
